pkg/cmd/server/api/types.go

```diff
@@ -474,6 +473,17 @@ type AuditConfig struct {
 	MaximumRetainedFiles int
 	// Maximum size in megabytes of the log file before it gets rotated. Defaults to 100MB.
 	MaximumFileSizeMegabytes int
+
+	// Path to the file that defines the audit policy configuration.
+	PolicyFile string
+
+	// Format of saved audits (legacy or json).
+	LogFormat string
+
+	// Path to a kubeconfig formatted file that defines the audit webhook configuration.
+	WebhookConfigFile string
```

Comment:
WebhookKubeconfigFile would be clearer.

Reply:
> WebhookKubeconfigFile would be clearer.

Our config uses KubeConfig, but as I recall there are special allowances for webhooks to set a termination url path, right?

Reply:
Yeah, but the file format follows KubeConfig, I've already applied that change.